Formula/a/ansible-lint.rb

```diff
@@ -19,34 +19,51 @@ class AnsibleLint < Formula
 
   depends_on "pkg-config" => :build
   depends_on "rust" => :build # for rpds-py
-  depends_on "ansible"
-  depends_on "black"
   depends_on "certifi"
```

Comment:
I was just about to make the same PR 😆 

Looks like this doesn't need `certifi` anymore